src/celeritas/user/detail/SDParams.hh

```diff
@@ -0,0 +1,59 @@
+//------------------------------- -*- C++ -*- -------------------------------//
+// Copyright Celeritas contributors: see top-level COPYRIGHT file for details
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/user/detail/SDParams.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "corecel/data/AuxInterface.hh"
+#include "corecel/data/Collection.hh"
+#include "corecel/data/CollectionMirror.hh"
+#include "corecel/data/ParamsDataInterface.hh"
+#include "corecel/io/Label.hh"
+
+#include "../SDData.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+class GeoParamsInterface;
+
+namespace detail
+{
```

Comment:
I think we can put the SD params in the top-level database, and in fact we may want to add it to core params.

Reply:
Not sure I understand this comment correctly. To clarify, remove the detail namespace here to keep the `SDParams` just in the `celeritas` namespace? Would I need to move the file to a different directory, like `global/detail` where the existing global `CoreParams` are? and when you say add it to Core params, do you mean add it to the global core params similar to how it is added to the optical version here already?

Reply:
Sorry, I meant to write top level *namespace*, so just move it to `celeritas/user`. In a follow-on commit, we might want to add this to the global core params.